app/views/users/edit.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= form_for(@user) do |f| %>
+  <div class="form-inputs">
+    <p><%= f.text_field :email %></p>
+    <p><%= f.select(:role, User.roles.keys.map {|role| [role.titleize,role]})%></p>
```

Comment:
I know that I am boring but please use styleguides :) `User.roles.keys.map{ |role| [role.titleize,role] }) %>`